app/package.json

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
Do we add some more infos about the project here? Like project name, github url, issue url etc.?